web/html/src/components/tree.js

```diff
@@ -0,0 +1,163 @@
+// @flow
+
+const { hot } = require('react-hot-loader');
```

Comment:
Can we move this to ES6 imports? at least in the reusable tree component. (The other would be nice as well)